app/Mail/MemberAppConfirmation.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Mail;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Mail\Mailable;
+use Illuminate\Mail\Mailables\Content;
+use Illuminate\Mail\Mailables\Envelope;
+use Illuminate\Queue\SerializesModels;
+
+class MemberAppConfirmation extends Mailable
```

Comment:
It makes sense to differentiate this mail class from the other MemberApp mail class but is there a better and less ambiguous name we could use. I think the name could be quite confusing given some time to forget and the potential to onboard additional developers.

Reply:
Do you have a suggestion of what to call it? I would have gone with MemberApplicationConfirmation, since I don't mind long names but that wouldn't have fit the existing pattern.

some other words could be: receipt, acknowledgement

let me know what you think

Reply:
Member[Application|App]InterviewSummary and Member[Application|App]InterviewConfirmation maybe?